charts/rstudio-workbench/Chart.yaml

```diff
@@ -1,8 +1,8 @@
 name: rstudio-workbench
 description: Official Helm chart for Posit Workbench
-version: 0.9.11
+version: 0.9.12
```

Comment:
Should we be bumping this up by a more major amount, or is that not helm kosher?

Reply:
Looks like the minor version is typically bumped if breaking changes to the chart itself are introduced, so I don't think that applies here.